docs/content/docs/connectors/models/openai.md

```diff
@@ -0,0 +1,244 @@
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                <h5>provider</h5>
+            </td>
+            <td>required</td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>String</td>
+            <td>Specifies the model function provider to use, must be 'openai'.</td>
+        </tr>
+        <tr>
+            <td>
+                <h5>endpoint</h5>
+            </td>
+            <td>required</td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>String</td>
+            <td>Full URL of the OpenAI API endpoint, e.g. <code>https://api.openai.com/v1/chat/completions</code> or
+                <code>https://api.openai.com/v1/embeddings</code>.</td>
+        </tr>
+        <tr>
+            <td>
+                <h5>api-key</h5>
+            </td>
+            <td>required</td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>String</td>
+            <td>OpenAI API key for authentication.</td>
+        </tr>
+        <tr>
+            <td>
+                <h5>model</h5>
+            </td>
+            <td>required</td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>String</td>
+            <td>Model name, e.g. <code>gpt-3.5-turbo</code>, <code>text-embedding-ada-002</code>.</td>
+        </tr>
+    </tbody>
+</table>
+
+### Chat Completions
+
+<table class="table table-bordered">
+    <thead>
+        <tr>
+            <th class="text-left" style="width: 25%">Option</th>
+            <th class="text-center" style="width: 8%">Required</th>
+            <th class="text-center" style="width: 7%">Default</th>
+            <th class="text-center" style="width: 10%">Type</th>
+            <th class="text-center" style="width: 50%">Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                <h5>system-prompt</h5>
+            </td>
+            <td>optional</td>
+            <td style="word-wrap: break-word;">"You are a helpful assistant."</td>
+            <td>String</td>
+            <td>The input message for the system role.</td>
+        </tr>
+        <tr>
+            <td>
+                <h5>temperature</h5>
+            </td>
+            <td>optional</td>
+            <td style="word-wrap: break-word;">null</td>
+            <td>Double</td>
+            <td>Controls randomness of output, range <code>[0.0, 1.0]</code>. See <a href=\"https://platform.openai.com/docs/api-reference/responses/create#responses-create-temperature\">temperature</a></td>
+        </tr>
+        <tr>
+            <td>
+                <h5>top-p</h5>
+            </td>
+            <td>optional</td>
+            <td style="word-wrap: break-word;">null</td>
+            <td>Double</td>
+            <td>Probability cutoff for token selection (used instead of temperature). See <a href=\"https://platform.openai.com/docs/api-reference/responses/create#responses-create-top_p\">top_p</a></td>
```

Comment:
Seems the current link doesn't work. You can take a look at this.

https://github.com/apache/flink-connector-kafka/blob/main/docs/content/docs/connectors/table/kafka.md?plain=1#L236C130-L236C183